### ISSUE TITLE
build failures with gcc5+?

### ISSUE THREAD

**Issue by karlp** [Neopallium/nixio#13]
Not sure what this is trying to do even

```
src/process.c: In function ‘nixio__exec’:
src/process.c:76:26: error: comparison of constant ‘4’ with boolean expression is always true [-Werror=bool-compare]
     if (!lua_type(L, -2) != LUA_TSTRING || !lua_isstring(L, -1)) {
                          ^
src/process.c:76:26: error: logical not is only applied to the left hand side of comparison [-Werror=logical-not-parentheses]
```

**Comment by karlp** [comment]
seems the first ! shouldn't be there, compared to openwrt luci sources at least.

**Comment by Neopallium** [comment]
Fix in release 0.4: `luarocks install Neopallium/nixio`
